mods/ctf/ctf_classes/melee.lua

```diff
@@ -105,7 +105,7 @@ minetest.register_tool("ctf_classes:sword_bronze", {
 		sword_special_timer[pname] = 20
 		sword_special_timer_func(pname, 20)
 
-		minetest.registered_chatcommands["m"].func(pname, "Marked with "..pname.."'s sword")
+		minetest.registered_chatcommands["m"].func(pname, ""..pname.."'s sword marker")
```

Comment:
```suggestion
		minetest.registered_chatcommands["m"].func(pname, ""..pname.."'marked with sword")
```

"player's sword marker" doesn't sound good.

Reply:
Here you forgot to remove '